csharp/extractor/Semmle.Util/FileUtils.cs

```diff
@@ -86,16 +86,32 @@ public static void TryDelete(string file)
         }
 
         /// <summary>
-        /// Computes the hash of <paramref name="filePath"/>.
+        /// Computes the hash of the file at <paramref name="filePath"/>.
         /// </summary>
         public static string ComputeFileHash(string filePath)
         {
             using var fileStream = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read);
-            using var shaAlg = SHA256.Create();
-            var sha = shaAlg.ComputeHash(fileStream);
+            var sha = SHA256.HashData(fileStream);
+            return GetHashString(sha);
+        }
+
+        /// <summary>
+        /// Computes the hash of <paramref name="input"/>.
+        /// </summary>
+        public static string ComputeHash(string input)
+        {
+            var bytes = Encoding.Unicode.GetBytes(input);
+            var sha = MD5.HashData(bytes); // MD5 to keep it shorter than SHA256
```

Comment:
I tried calculating the propability of collision for 1k elements on a 128bit hash - the calculator rounded it to 0 😂